api/app/wildfire_one.py

```diff
@@ -368,14 +360,15 @@ async def fetch_hourlies(
         session: ClientSession,
         raw_station: dict,
         headers: dict,
-        time_of_interest: datetime) -> WeatherStationHourlyReadings:
+        start_timestamp: int,
+        end_timestamp: int) -> WeatherStationHourlyReadings:
     """ Fetch hourly weather readings for the past 5 days for a give station
     TODO: rename this function, or specify the time range.
```

Comment:
looks to me like we can now delete this TODO, since you are now specifying the time range